pkg/registry/apis/iam/user/rest_display.go

```diff
@@ -146,25 +149,21 @@ func parseKeys(req []string) dispKeys {
 			switch t {
 			case claims.TypeAnonymous:
 				keys.disp = append(keys.disp, iamv0.IdentityDisplay{
-					IdentityType: t,
-					Display:      "Anonymous",
-					AvatarURL:    dtos.GetGravatarUrl(fakeCfgForGravatar, string(t)),
-				})
-				continue
-			case claims.TypeAPIKey:
-				keys.disp = append(keys.disp, iamv0.IdentityDisplay{
-					IdentityType: t,
-					UID:          key,
-					Display:      "API Key",
-					AvatarURL:    dtos.GetGravatarUrl(fakeCfgForGravatar, string(t)),
```

Comment:
I am not sure we need to keep APIKeys. We no longer support creating new ones and will pretty soon migrate them all to service accounts

Reply:
given that it is just an output conversion, I don't think it hurts to keep the value.

We can remove it when we remove `claims.TypeAPIKey`